.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+image: gitpod/workspace-node:2022-07-29-14-59-15
```

Comment:
Can we write this like this, so we avoid having to maintain the image version?

```yaml
image: gitpod/workspace-node:latest
```

Reply:
> Can we write this like this, so we avoid having to maintain the image version?
> 
> ```yaml
> image: gitpod/workspace-node:latest
> ```

I'm following the recommended practices here https://www.gitpod.io/docs/config-docker#docker-image-tags can be done if you prefer it.

Reply:
The template is dev-oriented and actively maintained (thanks to contributors like you!), so it should be fine to use `latest`.